scIB/metrics/graph_connectivity.py

```diff
@@ -26,6 +26,6 @@ def graph_connectivity(adata, label_key):
             connection='strong'
         )
         tab = pd.value_counts(labels)
-        clust_res.append(tab[0] / sum(tab))
+        clust_res.append(tab.max() / sum(tab))
```

Comment:
should this be argmax or max?

Reply:
max, if we want the number of nodes in the largest connected component, argmax if we want its label